lib/active_model/serializer.rb

```diff
@@ -5,7 +5,7 @@
 require 'active_model/serializer/configuration'
 require 'active_model/serializer/fieldset'
 require 'active_model/serializer/lint'
-require 'active_model/serializer/utils'
+require 'active_model/serializer/include_tree'
```

Comment:
Not a blocker for this PR, but I don't find `IncludeTree` an immediately obvious  name->responsibility.  Maybe `IncludedAttributesTree`, which makes it sound like it should be in the `serializer` folder, maybe